Assets/Mirror/Runtime/Transport/MultiplexTransport.cs

```diff
@@ -51,18 +49,6 @@ public override bool Available()
         }
 
         #region Client
-        // clients always pick the first transport
```

Comment:
should be a separate pr

Reply:
InitClient is removed because it we can't add handler in awake